generated/_task_modules/channels/ai_channel.py

```diff
@@ -0,0 +1,202 @@
+# Do not edit this file; it was automatically generated.
```

Comment:
This file is generated using the sample metadata which contains only few of the attributes present in this class.

Reply:
When you use the full metadata, does this output attributes in the same order as before? If not, this is going to be hard to validate without fixing the order.

Reply:
Yes Brad, the order of the generated attributes remains similar to the current code. We'll make sure that there is no discrepancy there.